tools/ci-build/publisher/src/subcommand/yank_release.rs

```diff
@@ -98,14 +142,10 @@ fn release_metadata(manifest: VersionsManifest) -> Result<Release> {
     }
 }
 
-fn confirm_plan(release: &Release) -> Result<()> {
-    let tag = release.tag.as_ref().ok_or_else(|| {
-        anyhow!("Versions manifest doesn't have a release tag. Can only yank tagged releases.")
-    })?;
-
+fn confirm_plan(tag: &str, crates: &BTreeMap<String, String>) -> Result<()> {
     info!("This will yank aws-sdk-rust's `{tag}` release from crates.io.");
     info!("Crates to yank:");
-    for (crate_name, crate_version) in &release.crates {
+    for (crate_name, crate_version) in crates {
         info!("   {}-{}", crate_name, crate_version);
```

Comment:
_Feel free to ignore this since you didn't touch this line._
```suggestion
        info!("\t{}-{}", crate_name, crate_version);
```
We should use tab characters instead of spaces for stuff like this.